docs/gaia/gaiacli.md

```diff
@@ -231,7 +233,12 @@ gaiacli tx send \
   --generate-only > unsignedSendTx.json
 ```
 
-You can now sign the transaction file generated through the `--generate-only` flag by providing your key to the following command:
+__Note__: Simulation cannot be used in conjunction with tx generation only
```

Comment:
Can we have a custom container to make the note bigger ? (check https://vuepress.vuejs.org/guide/markdown.html#custom-containers)

Reply:
And why is the simulation not working in this case?

Reply:
@fedekunze yeah sure, updated!

@faboweb not sure what you mean? Is the comment/note not explanatory enough? You can't use `simulate` and `generate_only` together because `simulate` requires a pubkey and `generate_only` does not use a Keybase (as requested) and thus you cannot attach a pubkey.

Reply:
Why does a simulation need a real pubkey?

Reply:
Is this the same issue: https://github.com/cosmos/cosmos-sdk/issues/3423 ?

Reply:
No, I cannot reproduce #3396 -- "auto" works for me. Simulation needs a public key because a large part of gas estimation is signature verification.

Reply:
ah ok, I missed that those are different things. But for signature verification you could use any pubkey, can't you?

Reply:
Let's not pollute this PR with the discussion and move it over to https://github.com/cosmos/cosmos-sdk/issues/3430